src/sentry/auth/superuser.py

```diff
@@ -42,7 +44,7 @@
 MAX_AGE = getattr(settings, "SUPERUSER_MAX_AGE", timedelta(hours=4))
 
 # the maximum time the session can stay alive without making another request
-IDLE_MAX_AGE = getattr(settings, "SUPERUSER_IDLE_MAX_AGE", timedelta(minutes=30))
+IDLE_MAX_AGE = getattr(settings, "SUPERUSER_IDLE_MAX_AGE", timedelta(minutes=15))
```

Comment:
did we all agree on this change? if more discussion needs to be had, may be worth putting this change in a separate PR.

Reply:
No, there wasn't any discussion. Although i feel like 15 mins would be better if we plan to limit privileged access 15 mins. What do u think? Ill ask danny as well and ill [move it](https://github.com/getsentry/sentry/pull/32104) depending on what we decide on